spec/integration/driver_spec.rb

```diff
@@ -56,6 +56,54 @@ def session_url(path)
       end
     end
 
+    def capture_stdout
```

Comment:
I think this can be replaced with RSpec's output matcher - https://www.relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/output-matcher